core/src/processing/opengl/PSurfaceJOGL.java

```diff
@@ -195,7 +195,8 @@ protected void initGL() {
     if (profile == null) {
       if (PJOGL.profile == 2) {
         try {
-          profile = GLProfile.getGL2ES1();
+          // request at least GL2 or GLES2
+          profile = GLProfile.getGL2ES2();
```

Comment:
So we'll need to hold on this until https://github.com/processing/processing/issues/3505